fsgs/Database.py

```diff
@@ -439,18 +439,11 @@ def get_variant_for_list_and_game(self, list_uuid, game_uuid):
             return None
         return row[0]
 
-    def find_games_new(self, search="", have=3, list_uuid="",
+    def find_games_new(self, search="", have=3, list_uuid=None,
```

Comment:
Since we are checking for `list_uuid` "Noneness", better to have None as default